supreme/src/commonMain/kotlin/at/asitplus/signum/supreme/sign/EphemeralKeys.kt

```diff
@@ -2,16 +2,23 @@ package at.asitplus.signum.supreme.sign
 
 import at.asitplus.KmmResult
 import at.asitplus.catching
+import at.asitplus.signum.indispensable.CryptoPrivateKey
 import at.asitplus.signum.indispensable.CryptoPublicKey
 import at.asitplus.signum.indispensable.Digest
 import at.asitplus.signum.indispensable.RSAPadding
 import at.asitplus.signum.indispensable.SignatureAlgorithm
 import at.asitplus.signum.indispensable.nativeDigest
+import at.asitplus.signum.supreme.SecretExposure
 import at.asitplus.signum.supreme.dsl.DSL
 import at.asitplus.signum.supreme.dsl.DSLConfigureFn
 import at.asitplus.signum.supreme.os.SignerConfiguration
 
+
+@SecretExposure
+internal expect fun EphemeralKeyBase<*>.exportPrivate(): CryptoPrivateKey<*>
 internal expect fun makeEphemeralKey(configuration: EphemeralSigningKeyConfiguration) : EphemeralKey
+internal expect fun  makePrivateKeySigner(key: CryptoPrivateKey.EC, algorithm: SignatureAlgorithm.ECDSA) : Signer.ECDSA
```

Comment:
should this not be `CryptoPrivateKey.signer`, or `Signer.for(privateKey)`, or something along those lines? ideally matching how we map a `CryptoPublicKey` to a `Verifier`